packages/components/src/tools-panel/styles.ts

```diff
@@ -145,3 +145,19 @@ export const ToolsPanelItemPlaceholder = css`
 export const DropdownMenu = css`
 	min-width: 200px;
 `;
+
+export const DefaultControlsItem = css`
+	span:last-child {
```

Comment:
Since we are the ones providing the "Reset" span, would it be possible to avoid a `span:last-child` selector and target it more explicitly? It's hard to tell what `span:last-child` is, and it's a bit fragile.

Reply:
We certainly can. In the `span:last-child` approach I was hoping to avoid plumbing through multiple dynamic classnames. I've pushed a different solution using a `styled.span` in https://github.com/WordPress/gutenberg/pull/44260/commits/93ca39d3552ae7ec10b8a6123bd4ff339998e9eb. Is that any better?

Reply:
Yep, thank you!